web3.js/test/epoch-schedule.test.ts

```diff
@@ -0,0 +1,32 @@
+import {expect} from 'chai';
+
+import {EpochSchedule} from '../src';
+
+describe('EpochSchedule', () => {
+  it('slot methods work', () => {
+    const firstNormalEpoch = 8;
+    const firstNormalSlot = 8160;
+    const leaderScheduleSlotOffset = 8192;
+    const slotsPerEpoch = 8192;
+    const warmup = false;
```

Comment:
I think it'd be better to use the actual defaults here to ensure that this implementation is correct
```suggestion
    const firstNormalEpoch = 14;
    const firstNormalSlot = 524_256;
    const leaderScheduleSlotOffset = 432_000;
    const slotsPerEpoch = 432_000;
    const warmup = true;
```